tests/contrib/aioredis/test_aioredis.py

```diff
@@ -22,7 +24,7 @@ async def redis_client():
 def get_redis_instance():
     if aioredis_version >= (2, 0):
         return aioredis.from_url("redis://127.0.0.1:%s" % REDIS_CONFIG["port"])
-    return aioredis.create_redis(("localhost", REDIS_CONFIG["port"]))
+    return aioredis.create_redis_pool(("127.0.0.1", REDIS_CONFIG["port"]))
```

Comment:
does this has any particular impact on the other tests?

instead should we introduce a new fixture for a `redis_pool` ?

at the very least we should add a fixture for an `empty_redis_pool` to handle properly setting it up and tearing it down:


```python
@pytest.mark.asyncio
@pytest.fixture
async def empty_redis_pool():
    client = aioredis.create_redis_pool()
    original_pool = getattr(client.connection, "_pool", None)
    if original_pool:
        client.connection._pool = collections.deque(maxlen=1)
    try:
        yield client
    finally:
        client.connection._pool = original_pool
```


What do you think?

Reply:
Good idea, that would solve the issue we have at the moment. As for the original fixture, it is creating a pool for aioredis 2+, so, it would make sense to do the same for aioredis 1.3 too. I will add a fixture for empty pool.